rascal-vscode-extension/src/lsp/ParameterizedLanguageServer.ts

```diff
@@ -41,7 +41,7 @@ export class ParameterizedLanguageServer implements vscode.Disposable {
         private readonly absoluteJarPath: string,
         private readonly deployMode = true,
         private readonly languageId = 'parametric-rascalmpl',
-        private readonly title = 'Language Parametric Rascal Language Server',
```

Comment:
This should be reverted. Or is there a reason this is happening @jurgenvinju ?

Reply:
Apparantly this has been reverted and the code in the comment is out of date.

Reply:
No, there is still a change in this line, that I don't think should be here.

Reply:
I've removed this change, lets see if the test sill succeed.